README.rst

```diff
@@ -62,6 +62,10 @@ settings file.
 Underscoreize Options
 =====================
 
+###########################
+No Underscore Before Number
+###########################
+
 As raised in https://github.com/krasa/StringManipulation/issues/8#issuecomment-121203018
```

Comment:
Not really related, but maybe just beautify the README a little with a "this comment"-clickable instead of the entire link printed out?

```suggestion
As raised in `this comment <https://github.com/krasa/StringManipulation/issues/8#issuecomment-121203018>`_
```